nullaway/src/test/java/com/uber/nullaway/NullAwayAndroidTest.java

```diff
@@ -0,0 +1,211 @@
+package com.uber.nullaway;
+
+import com.google.errorprone.CompilationTestHelper;
+import com.sun.tools.javac.main.Main;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link com.uber.nullaway.NullAway}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("CheckTestExtendsBaseClass")
+public class NullAwayAndroidTest {
+  @Rule public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  private CompilationTestHelper compilationHelper;
+
+  @Before
+  public void setup() {
+    compilationHelper = CompilationTestHelper.newInstance(NullAway.class, getClass());
+    compilationHelper.setArgs(
+        Arrays.asList(
+            "-d",
+            temporaryFolder.getRoot().getAbsolutePath(),
+            "-XepOpt:NullAway:KnownInitializers="
```

Comment:
Do we really need all these options for these tests?

Reply:
Not really. Removed most of them